spec/unit/crypto/secrets.spec.ts

```diff
@@ -81,26 +76,6 @@ describe("Secrets", function () {
         return global.Olm.init();
     });
 
-    it("should allow storing a default key", async function () {
```

Comment:
unlike the rest of the tests in this file, this one instantiates its own SecretStorage and uses that, rather than going via the instance in the test client. In fact, I'd say that all the rest of the tests are much closer integration tests, despite the name of the file.

Anyway, this test is not like the others, so is a good candidate for moving. The rest need sorting out another time.